app.rb

```diff
@@ -14,6 +15,12 @@ def initialize
     @people = []
   end
 
+  def greet
+    puts 'Welcome to the School Library App!'
+    puts "\n"
+    menu
+  end
+
   # list all books
   def list_books
     if @books.empty?
```

Comment:
- [ ] Kindly separate functions in `app.rb` in several classes as the Single-Responsibility Principle of the SOLID object Oriented Design states that a class should have one job only. hence all requests for inputs(`create_person`,`create_student`,`create_teacher`,`create_book`,`create_rental`) should have been placed in the separate files whose responsibilities are to request input. Please kindly work on this and move a step forward toward approval.

For more on the SOLID lessons, click on this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid)

Reply:
Thank you for the review. Fixed it now.